pinot-common/src/main/java/org/apache/pinot/common/datablock/DataBlockUtils.java

```diff
@@ -230,6 +230,28 @@ public static ByteString toByteString(DataBlock dataBlock)
     return byteString;
   }
 
+  public static List<ByteString> toByteStrings(DataBlock dataBlock, int maxBlockSize)
+      throws IOException {
+    List<ByteBuffer> bytes = dataBlock.serialize();
+    if (bytes.isEmpty()) {
+      return List.of(ByteString.EMPTY);
```

Comment:
Why not just an empty list? In case this is actually necessary, I think it should be added in this method javadoc